rubicon/objc/api.py

```diff
@@ -649,6 +656,38 @@ def __new__(cls, object_ptr, _name=None, _bases=None, _ns=None):
 
         return self
 
+    def release(self):
+        """
+        Manually decrement the reference count of the corresponding objc object
+
+        The objc object is sent a dealloc message when its reference count reaches 0. Calling
+        this method manually should not be necessary, the object will be automatically
+        released when the Python object is deallocated.
```

Comment:
This should be updated now that an explicit `release` is necessary again when the object was explicitly `retain`ed before.

```suggestion
        The objc object is sent a dealloc message when its reference count reaches 0. Calling
        this method manually should not be necessary, unless the object was explicitly
        ``retain``\ed before. Objects returned from ``.alloc().init...(...)`` and similar calls
        are released automatically by Rubicon when the corresponding Python object is deallocated.
```